### DISCUSSION TITLE
How to display top event first

### DISCUSSION THREAD

**Discussion by jkkoenig** [derhansen/sf_event_mgt#1227] [Q&A]
In Typo3 11 sf_event_mgt extension 6.4.1., I can display either top events or all events except top events.

I would like to display the top events first and then directly followed by other events (like the blog "featured post" in Wordpress)

I tried a if-condition in Resources/Private/Templates/Event/list.html but it does not display the top events:

```
   <div class="row">
    <f:for each="{events}" as="event">
       <f:if condition="{event.top_event}">
        <f:render partial="Event/TerminlisteGridStartseite" arguments="{_all}"/>
       </f:if> 
    </f:for>
    <f:for each="{events}" as="event">
        <f:render partial="Event/TerminlisteGridStartseite" arguments="{_all}"/>
    </f:for>
    </div>

```
Is there a way to achieve this?

**Comment by derhansen** [comment]
This is not possible currently, but I can implement this feature (see #1228), so top events will always be shown on top of the list regardless of all other sorting options.

**Comment by jkkoenig** [comment]
This would be very useful, thank you!

 

Von: Torben Hansen ***@***.***> 
Gesendet: Donnerstag, 25. April 2024 19:50
An: derhansen/sf_event_mgt ***@***.***>
Cc: jkkoenig ***@***.***>; Author ***@***.***>
Betreff: Re: [derhansen/sf_event_mgt] How to display top event first (Discussion #1227)

 

This is not possible currently, but I can implement this feature (see #1228 <https://github.com/derhansen/sf_event_mgt/issues/1228> ), so top events will always be shown on top of the list regardless of all other sorting options.
